Makefile

```diff
@@ -575,8 +589,30 @@ bin/test-connection: $(SRC_FILES) vendor/.up-to-date
 	    sh -c 'go build -v -i -o $@ -v $(LDFLAGS) "$(PACKAGE_NAME)/fv/test-connection"'
 
 ###############################################################################
-# CI
+# CI/CD
 ###############################################################################
+.PHONY: ci cd
+
+## run CI cycle - build, test, etc.
+ci: image bin/calico-felix.exe ut upload-to-coveralls static-checks
+ifeq (,$(filter fv, $(EXCEPT)))
+	@$(MAKE) fv
+endif
+ifeq (,$(filter k8sfv-test, $(EXCEPT)))
+	@$(MAKE) k8sfv-test JUST_A_MINUTE=true USE_TYPHA=true
+	@$(MAKE) k8sfv-test JUST_A_MINUTE=true USE_TYPHA=false
+endif
+
+## Deploy images to registry
+cd:
+ifndef CONFIRM
+	$(error CONFIRM is undefined - run using make <target> CONFIRM=true)
+endif
+ifndef BRANCH_NAME
+	$(error BRANCH_NAME is undefined - run using make <target> BRANCH_NAME=var or set an environment variable)
+endif
+	$(MAKE) tag-images push IMAGETAG=${BRANCH_NAME}
```

Comment:
I think you need `$(BRANCH_NAME)` here instead of `${BRANCH_NAME}`

Reply:
Fixing.